<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QComboBox" name="manBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listManDream">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>256</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="womanBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>20</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>SELECT a MAN</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>THE LIST OF DESIRABLE WOMEN</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>SELECT A WOMAN</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWomanDream">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>250</y>
     <width>256</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>THE LIST OF DESIRABLE MEN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="show_coupes">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>550</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Show_possible_couples </string>
   </property>
  </widget>
  <widget class="QPushButton" name="man_info">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>550</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>See_info_about_men</string>
   </property>
  </widget>
  <widget class="QPushButton" name="women_info">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>550</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>See_info_about_women</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_woman">
   <property name="geometry">
    <rect>
     <x>664</x>
     <y>10</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>add_new_woman</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_man">
   <property name="geometry">
    <rect>
     <x>664</x>
     <y>50</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>add_new_man</string>
   </property>
  </widget>
  <widget class="QPushButton" name="del_woman">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>120</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>delete_woman</string>
   </property>
  </widget>
  <widget class="QPushButton" name="del_man">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>160</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>delete_man</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_man">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>220</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit_a_man</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_woman">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>260</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit_a_woman</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
